lib/teleterm/apiserver/handler/handler_access_requests.go

```diff
@@ -16,10 +16,12 @@ package handler
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/gravitational/trace"
 	"google.golang.org/protobuf/types/known/timestamppb"
 
+	"github.com/gravitational/teleport/e/lib/web/ui"
```

Comment:
I'm not sure this is going to work. To date, no teleport code depends on e. Can we maintain that property?

Reply:
This was an oversight. I even mentioned that I didn't want to depend on `e` and brought this in. My mistake!  Will update